### DISCUSSION TITLE
Welcome to portfolio Discussions!

### DISCUSSION THREAD

**Discussion by tresamingo** [three-amingos/portfolio#1] [General]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by tresamingo** [comment]
This is the beginning of from the future @2021

**Comment by basit82002** [comment]
Hello Amingo!

**Comment by basit82002** [comment]
Let's discuss any code related discussions here please
